go.mod

```diff
@@ -30,7 +30,8 @@ require (
 	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
 	github.com/aws/aws-lambda-go v1.13.3
 	github.com/aws/aws-sdk-go v1.38.60
-	github.com/aws/aws-sdk-go-v2 v0.24.0
+	github.com/aws/aws-sdk-go-v2 v0.26.0
+	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v0.26.0
```

Comment:
[service/cloudwatchlogs/v1.15.5](https://github.com/aws/aws-sdk-go-v2/releases/tag/service%2Fcloudwatchlogs%2Fv1.15.5) is the latest?